README.md

```diff
@@ -1 +1,45 @@
-# fancy-title-generator
+# Fancy Fashion Product Title Generator
+
+Fancy title generator is a web application that can help you to generate a title for your fashion products.
+
+## Prerequisites
+
+Before you get started, make sure you met the following requirements:
+* Python 3.10.2
+* git
+* node.js(npm)
```

Comment:
Need to lock the node version too. [asdf](https://asdf-vm.com/) is one of the tools to manage versions for basically everything. You can use it in your project for node and python version locking.

Node version could be the latest LTS release - check the exact version in [here](https://nodejs.org/en/).
For python it's convenient to use the first patch of the current major version - right now it's 3.10.1

Then you can simply refer to asdf tool versions file so you don't have to change the readme once you decide to upgrade.

No need to mention git - assumed since you are on Github.